### PR TITLE
Rework README.md #48 a start

### PR DESCRIPTION
**Pull request by olavea**
I want this README to be easier to get started with:

1. A shorter intro with:
2. What is this?
3. Why: what is the number 1 pain you have before you start using this plugin
4. Promise: faster loads?
5. How: Quickly get to the how to install and start using 
6. Do we need a table of contents?
7. Question: How do your devs test this manually? 
8. Question: How do you test the performance increase? Lghthouse? Web vitals?

### CODE REVIEW

**Comment by raae** [comment]
The readme should not be about selling Cloudinary as a service, it should be to help a user use the plugin.

File: README.md [hunk 1]
```diff
@@ -177,74 +177,73 @@ export const query = graphql`
...
 ### `apiKey` (required)
 
 The API Key of your Cloudinary account. You'll find it in your [Cloudinary console](https://cloudinary.com/console/).
 
-**Type:** String  
-**Default:** n/a  
-**Note:** This should be stored and retrieved as an environment variable.
+**Type:** String
+**Default:** n/a
+**Note:** Store and retrieve your `apiKey` as an environment variable.
```
Comment (raae) [inline README.md]: Add link to env variable documentation

File: README.md [hunk 2]
```diff
@@ -177,74 +177,73 @@ export const query = graphql`
...
 The _resource_ types to include when pulling data from Cloudinary.
 
-**Type:** String  
-**Default:** `image`  
-**Valid:** `image`, `raw` and `video`  
-**Note:** Use the video resource type for all video and audio files, such as `.mp3` and `.mp4`.
+**Type:** String
+**Default:** `image`
+**Valid:** `image`, `raw` and `video`
+**Note:** Use the video`resourceType` for all video and audio files, such as `.mp3` and `.mp4`.
```
Comment (raae) [inline README.md]: missing space

File: README.md [hunk 3]
```diff
@@ -177,74 +177,73 @@ export const query = graphql`
...
 
 ### `resourceType`
 
 The _resource_ types to include when pulling data from Cloudinary.
 
-**Type:** String  
-**Default:** `image`  
-**Valid:** `image`, `raw` and `video`  
-**Note:** Use the video resource type for all video and audio files, such as `.mp3` and `.mp4`.
+**Type:** String
```
Comment (raae) [inline README.md]: You have removed the double space at the end of all of these, removing the line break.

**Review by raae — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (olavea) [reply README.md]: I added the double space and prettier or something deleted them all when I saved. Can you add them?

File: README.md (hunk 2 above)
Comment (olavea) [reply README.md]: ✅

**Pull request closed without merging**